nevooronni/Politico_web_app
# #163600519 A user can signup

#### What does this PR do?
This is a PR for a feature that allows a user to signup and use the app

#### Description of Task to be completed?
The following are the tasks to be completed
* add html for signup page
* html markup with css

#### How to manually test feature?
1. clone this repo
``https://github.com/nevooronni/Politico-Andela.git```

2. cd in folder
```cd Politico-Andela```

3 switch to feature branch 
```git checkout ft-user-signup-163600519```

4. Open UI/html/index.html in your browser

5. Click the signup button on landing page
6. Should be able to view signup page where you can enter your details and signup

#### Pivotal Tracker story associated with this PR:
story id: #163600519

#### Screenshots
![screenshot from 2019-01-31 04-35-49](https://user-images.githubusercontent.com/20374625/52024414-d9303e00-2511-11e9-9015-8c2c69aff997.png)


![screenshot from 2019-01-31 04-35-59](https://user-images.githubusercontent.com/20374625/52024444-e64d2d00-2511-11e9-9233-f0f208253867.png)
